osu.Game/Overlays/MusicController.cs

```diff
@@ -150,7 +150,7 @@ public void EnsurePlayingSomething()
         {
             if (IsUserPaused) return;
 
-            if (CurrentTrack.IsDummyDevice)
+            if (CurrentTrack.IsDummyDevice || beatmap.Value.BeatmapSetInfo.DeletePending)
```

Comment:
Note that I've added this for future usage/safety. It's not required or used in this PR due to the reason mentioned in the inline comment.